src/Checkbox/checkbox.style.ts

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import { Colors } from "..";
+import mapping from "../../public/fonts/vivy-icons/iconMapping.json";
+
+const checkboxEmpty = mapping["icon-checkbox-empty"];
+const checkboxFilled = mapping["icon-checkbox-filled"];
+
+export const CheckBoxWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const CheckMark = styled.label`
+  padding: 0;
+  font-size: 24px;
```

Comment:
I don't know if we paid attention to the fact in other components in the past, don't look like it ... but could u change the px values to rem?
Also - we are using the 62.5% => 1.6rem trick in GlobalStyles so u can just change for example 24px to 2.4rem.

Reply:
I did discuss the lack of design tokens with @a7madgamal and I have been told that we only copy values for now?! I think it makes sense in general to create a system to re-use design tokens for fonts and spacing rather than re-writing styles. But not sure if this should be done in the scope of this ticket or if we should have a general cleanup?

Reply:
Also, did we implement the global styles trick also on the other repos that use the legacy version of this component? If not, then this would lead to a different font size than intended...

Reply:
Right, so after feedback from @sblaettgen and @hamzahsn I changed the size unit to REM. Will keep it in mind for further migrations, thanks guys 🙏

Reply:
@fabwoofer asked me about this in Slack ... it is just plain wrong to use `px` because the globals are setting everything up for REM even in the legacy code. My guess is that it was overlooked or rushed or whatever but they are (should -provided that the global styles are used correctly) even in legacy (JS) fully supported.

So I strongly voted for `rem` here ... it really is a bugfix 😄

Reply:
> Right, so after feedback from @sblaettgen and @hamzahsn I changed the size unit to REM. Will keep it in mind for further migrations, thanks guys 🙏

Actually we have to thank you for your great work 🙏  😄